podman/tests/integration/base.py

```diff
@@ -0,0 +1,46 @@
+#   Copyright 2020 Red Hat, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+"""Base integration test code"""
+
+
+import shutil
+import uuid
+import os
+import fixtures
+import podman.tests.integration.utils as utils
+
+
+class IntegrationTest(fixtures.TestWithFixtures):
+    """Base Integration test case"""
+
+    def setUp(self):
+        super().setUp()
+
+        command = os.environ.get("PODMAN_BINARY", "podman")
+        if shutil.which(command) is None:
+            self.skipTest(f"{command} is not installed, skipping integration tests")
```

Comment:
I'm assuming a test-skip will ultimately result in a *PASS*.  Just as a warning: This can lead to a out-of-sight-out-of-mind situation at a high-level.  For example, PR's getting merged because "The tests are all green" where in reality, they're being skipped and nobody looked.  Granted may be a a corner-case, but it might be worth considering having this fail instead of skip.  If higher-layers shouldn't be running integration tests, they can exclude it at the harness-level more obviously/explicitly, possibly limiting surprises.